kubernetes/cray-opa/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-opa
-version: 1.32.9
+version: 1.32.10
```

Comment:
Would this not be a case to do a major bump? Changing from Deployment to Daemonset is a large change.